CODEOFCONDUCT.md

```diff
@@ -0,0 +1,55 @@
+# Code of Conduct
+
+All contributors and maintainers of this project are subject to this code of conduct.
+
+We pledge to respect everyone who contributes to the *Falcon* project or other associated activities by (including but not limited to) creating project issues, submitting pull requests, and providing feedback on the same. We also pledge to respect everyone who participates in discussions on the project's mailing list and IRC channel.
+
+Unacceptable behavior includes offensive verbal comments related to gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion, technology choices, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention.
```

Comment:
"includes but is not limited to" is a good phrase to use in these situations as you don't then have to list out each individual one, just the most common, or ones you want to specifically highlight.